<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFormPlayCtrl</class>
 <widget class="QWidget" name="CFormPlayCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CFormPlayCtrl</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame{
	/*border-image: url(:/Skin/bg-below.png);
	background-color: rgb(170, 0, 0);
background-color: rgb(100, 100, 100);*/
}

QLabel
{	border-image:none;
	color: rgb(255, 255, 255);
	font-size:15px;
	font-family:Microsoft Yahei;	
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="horizontalSpacing">
             <number>12</number>
            </property>
            <item row="0" column="6">
             <widget class="QPushButton" name="btnStepNext">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>单帧前进</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/stepNext.png)0 0  repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/stepNext.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/stepNext.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/stepNext.png)90 0 0 0 repeat
}</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="btnPrevious">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>上一个视频</string>
              </property>
              <property name="styleSheet">
               <string notr="true">/* top right bottom left*/
QPushButton
{
	
	border-image: url(:/Skin/control-button-previous.png)0 0  repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/control-button-previous.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/control-button-previous.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/control-button-previous.png)90 0 0 0 repeat
}</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="btnPause">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="toolTip">
               <string>播放</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/control-button-play-pause.png)0 0 repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/control-button-play-pause.png)40 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/control-button-play-pause.png)80 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/control-button-play-pause.png)80 0 0 0 repeat
}</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="btnStop">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>停止</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/control-button-stop.png)0 0  repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/control-button-stop.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/control-button-stop.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/control-button-stop.png)90 0 0 0 repeat
}</string>
              </property>
             </widget>
            </item>
            <item row="0" column="7">
             <widget class="QPushButton" name="btnNext">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>下一个视频</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/control-button-next.png)0 0  repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/control-button-next.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/control-button-next.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/control-button-next.png)90 0 0 0 repeat
}</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="btnSlow">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>慢进</string>
              </property>
              <property name="styleSheet">
               <string notr="true">/* top right bottom left*/
QPushButton
{
	
	border-image: url(:/Skin/control-button-rewind.png)0 0  repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/control-button-rewind.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/control-button-rewind.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/control-button-rewind.png)90 0 0 0 repeat
}</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QPushButton" name="btnFast">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>快进</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/control-button-fastforward.png)0 0  repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/control-button-fastforward.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/control-button-fastforward.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/control-button-fastforward.png)90 0 0 0 repeat
}</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="btnStepBack">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>单帧后退</string>
              </property>
              <property name="styleSheet">
               <string notr="true">/* top right bottom left*/
QPushButton
{
	
	border-image: url(:/Skin/stepBack.png)0 0  repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/stepBack.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/stepBack.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/stepBack.png)90 0 0 0 repeat
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="1">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="6">
           <layout class="QFormLayout" name="formLayout_5">
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="1" column="0">
             <widget class="QPushButton" name="btnClock">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>时间校准</string>
              </property>
              <property name="styleSheet">
               <string notr="true">
QPushButton
{
	
	border-image: url(:/Skin/clock.png)
}

QPushButton:hover
{	
	background-color: rgb(94, 94, 94);

}
QPushButton:pressed
{	
	
	background-color: rgb(65, 65, 65);
}
QPushButton:disabled
{	
	
}</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="4">
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="1">
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="5">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="horizontalSpacing">
             <number>5</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QPushButton" name="btnSound">
              <property name="minimumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="toolTip">
               <string>静音 开|关</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/sound.png)0 0  repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/sound.png)28 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/sound.png)56 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/sound.png)84 0 0 0 repeat
}</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="CMySlider" name="SliderVolume">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>10</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>10</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QSlider::groove:horizontal {
    border: 0px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
  /*  background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A1A1A1, stop:1 #B4B4B4);
	background-color: rgb(200, 200, 200);*/
	border-image:url(:/Skin/progress-base.png);
    margin: 2px 0 2px;
}

QSlider::handle:horizontal {
	border-image: url(:/Skin/thum.png);

    width: 16px;
 	height: 12px;
    margin: -3px 0px -7px 0px; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */

}

QSlider::sub-page:horizontal{ 	
	height: 4px;
	/*background-color: rgb(170, 0, 0);*/
border-image:url(:/Skin/progress-bar.png);
	margin: 2px 0px 2px 0px;
}s</string>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::NoTicks</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="QLabel" name="labelEndTime">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>00:00:00</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelStartTime">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>22</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel
{	border-image:none;
	color: rgb(255, 255, 255);
	font-size:15px;
	font-family:Microsoft Yahei;	
}</string>
            </property>
            <property name="text">
             <string>00:00:00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="CMySlider" name="SliderPlaytime">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>10</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::WheelFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QSlider::groove:horizontal {
    border: 0px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
  /*  background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A1A1A1, stop:1 #B4B4B4);
	background-color: rgb(200, 200, 200);*/
	border-image:url(:/Skin/progress-base.png);
    margin: 2px 0 2px;
}

QSlider::handle:horizontal {
	border-image: url(:/Skin/thum.png);

    width: 16px;
 	height: 12px;
    margin: -3px 0px -7px 0px; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */

}

QSlider::sub-page:horizontal{ 	
	height: 4px;
	/*background-color: rgb(170, 0, 0);*/
border-image:url(:/Skin/progress-bar.png);
	margin: 2px 0px 2px 0px;
}s</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="tracking">
             <bool>true</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CMySlider</class>
   <extends>QSlider</extends>
   <header>cmyslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="sisecplayer.qrc"/>
 </resources>
 <connections/>
</ui>
